<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>810</width>
    <height>662</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Super Searcher</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #2c2f33</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="superSearcher">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>160</y>
      <width>331</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Bebas Neue</family>
      <pointsize>48</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
      <underline>false</underline>
     </font>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="whatsThis">
     <string/>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 48pt &quot;Bebas Neue&quot;;
color: rgb(255, 114, 0);</string>
    </property>
    <property name="text">
     <string>SUPERSEARCHER</string>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="queryInput">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>250</y>
      <width>511</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(64, 68, 72);
font: 32pt &quot;Oswald&quot;;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="frame">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>250</y>
      <width>141</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 20pt &quot;Bebas Neue&quot;;
color: rgb(255, 114, 0);</string>
    </property>
    <property name="text">
     <string>Search</string>
    </property>
   </widget>
   <widget class="QPushButton" name="settings">
    <property name="geometry">
     <rect>
      <x>772</x>
      <y>0</y>
      <width>31</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../Downloads/Icons/Settings_Inactive.svg</normaloff>../Downloads/Icons/Settings_Inactive.svg</iconset>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="darkMode">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>0</y>
      <width>31</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset theme="darkModeTheme">
      <normaloff>../Downloads/Icons/Darkmode_Inactive.svg</normaloff>
      <normalon>../Downloads/Icons/Darkmode_DMode.svg</normalon>../Downloads/Icons/Darkmode_Inactive.svg</iconset>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QGroupBox" name="historySection">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>86</x>
      <y>530</y>
      <width>651</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhNone</set>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
    <widget class="QRadioButton" name="radioYes">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>0</y>
       <width>95</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QRadioButton
            {
            font : 32px 'Bebas Neue';
            color : rgb(64, 68, 72);
            }
QRadioButton::checked
            {
            color : rgb(255, 114, 0);
            }</string>
     </property>
     <property name="text">
      <string>Yes</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioNo">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>0</y>
       <width>81</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QRadioButton
            {
            font : 32px 'Bebas Neue';
            color : rgb(64, 68, 72);
            }
QRadioButton::checked
            {
            color : rgb(255, 114, 0);
            }</string>
     </property>
     <property name="text">
      <string>No</string>
     </property>
    </widget>
    <widget class="QLabel" name="coverUp1_6">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>20</y>
       <width>21</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="coverUp1_7">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>20</y>
       <width>20</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="superSearcher_4">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>-10</y>
       <width>651</width>
       <height>81</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Bebas Neue</family>
       <pointsize>24</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
       <underline>false</underline>
      </font>
     </property>
     <property name="mouseTracking">
      <bool>false</bool>
     </property>
     <property name="whatsThis">
      <string/>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 24pt &quot;Bebas Neue&quot;;
color: rgb(255, 114, 0);</string>
     </property>
     <property name="text">
      <string>Store History:</string>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_2">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>30</y>
       <width>151</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 20pt &quot;Bebas Neue&quot;;
color: rgb(255, 114, 0);</string>
     </property>
     <property name="text">
      <string>confirm</string>
     </property>
    </widget>
    <zorder>superSearcher_4</zorder>
    <zorder>radioYes</zorder>
    <zorder>radioNo</zorder>
    <zorder>coverUp1_6</zorder>
    <zorder>coverUp1_7</zorder>
    <zorder>pushButton_2</zorder>
   </widget>
   <widget class="QWidget" name="tabsSection" native="true">
    <property name="geometry">
     <rect>
      <x>86</x>
      <y>380</y>
      <width>651</width>
      <height>61</height>
     </rect>
    </property>
    <widget class="QRadioButton" name="radioButton3">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>10</y>
       <width>111</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QRadioButton
            {
            font : 48px 'Bebas Neue';
            color : rgb(64, 68, 72);
            }
QRadioButton::checked
            {
            color : rgb(255, 114, 0);
            }</string>
     </property>
     <property name="text">
      <string>3</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton1">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>10</y>
       <width>111</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QRadioButton
            {
            font : 48px 'Bebas Neue';
            color : rgb(64, 68, 72);
            }
QRadioButton::checked
            {
            color : rgb(255, 114, 0);
            }</string>
     </property>
     <property name="text">
      <string>1</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton2">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>10</y>
       <width>61</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QRadioButton
            {
            font : 48px 'Bebas Neue';
            color : rgb(64, 68, 72);
            }
QRadioButton::checked
            {
            color : rgb(255, 114, 0);
            }</string>
     </property>
     <property name="text">
      <string>2</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton5">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>10</y>
       <width>111</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QRadioButton
            {
            font : 48px 'Bebas Neue';
            color : rgb(64, 68, 72);
            }
QRadioButton::checked
            {
            color : rgb(255, 114, 0);
            }</string>
     </property>
     <property name="text">
      <string>5</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="superSearcher_3">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>651</width>
       <height>71</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Bebas Neue</family>
       <pointsize>24</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
       <underline>false</underline>
      </font>
     </property>
     <property name="mouseTracking">
      <bool>false</bool>
     </property>
     <property name="whatsThis">
      <string/>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 24pt &quot;Bebas Neue&quot;;
color: rgb(255, 114, 0);</string>
     </property>
     <property name="text">
      <string># of Tabs to open:</string>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton4">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>10</y>
       <width>61</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QRadioButton
            {
            font : 48px 'Bebas Neue';
            color : rgb(64, 68, 72);
            }
QRadioButton::checked
            {
            color : rgb(255, 114, 0);
            }</string>
     </property>
     <property name="text">
      <string>4</string>
     </property>
    </widget>
    <widget class="QLabel" name="coverUp1_8">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>20</y>
       <width>16</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="coverUp1_10">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>20</y>
       <width>16</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="coverUp1_9">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>20</y>
       <width>20</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="coverUp1_12">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>20</y>
       <width>16</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="coverUp1_11">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>20</y>
       <width>16</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <zorder>superSearcher_3</zorder>
    <zorder>radioButton3</zorder>
    <zorder>radioButton1</zorder>
    <zorder>radioButton2</zorder>
    <zorder>radioButton5</zorder>
    <zorder>radioButton4</zorder>
    <zorder>coverUp1_8</zorder>
    <zorder>coverUp1_10</zorder>
    <zorder>coverUp1_9</zorder>
    <zorder>coverUp1_12</zorder>
    <zorder>coverUp1_11</zorder>
   </widget>
   <widget class="QLabel" name="superSearcher_5">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>450</y>
      <width>631</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Bebas Neue</family>
      <pointsize>24</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
      <underline>false</underline>
     </font>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="whatsThis">
     <string/>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 24pt &quot;Bebas Neue&quot;;
color: rgb(255, 114, 0);</string>
    </property>
    <property name="text">
     <string>Allowed Domains:</string>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>390</y>
      <width>651</width>
      <height>211</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #2c2f33;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="errorText">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>120</y>
      <width>371</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font : 24px 'Bebas Neue';
color : rgb(255, 114, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>320</y>
      <width>471</width>
      <height>81</height>
     </rect>
    </property>
    <widget class="QRadioButton" name="radioButtonB">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>20</y>
       <width>81</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QRadioButton
            {
            font : 24px 'Bebas Neue';
            color : rgb(64, 68, 72);
            }
QRadioButton::checked
            {
            color : rgb(255, 114, 0);
            }</string>
     </property>
     <property name="text">
      <string>Bing</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButtonD">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>20</y>
       <width>121</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QRadioButton
            {
            font : 24px 'Bebas Neue';
            color : rgb(64, 68, 72);
            }
QRadioButton::checked
            {
            color : rgb(255, 114, 0);
            }</string>
     </property>
     <property name="text">
      <string>DuckDuckGo</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButtonG">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>20</y>
       <width>81</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QRadioButton
            {
            font : 24px 'Bebas Neue';
            color : rgb(64, 68, 72);
            }
QRadioButton::checked
            {
            color : rgb(255, 114, 0);
            }</string>
     </property>
     <property name="text">
      <string>Google</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButtonGS">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>20</y>
       <width>151</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QRadioButton
            {
            font : 24px 'Bebas Neue';
            color : rgb(64, 68, 72);
            }
QRadioButton::checked
            {
            color : rgb(255, 114, 0);
            }</string>
     </property>
     <property name="inputMethodHints">
      <set>Qt::ImhNone</set>
     </property>
     <property name="text">
      <string>Google Scholar</string>
     </property>
    </widget>
    <widget class="QLabel" name="coverUp1">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>20</y>
       <width>16</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="coverUp2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>20</y>
       <width>20</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="coverUp3">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>30</y>
       <width>20</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="coverUp4">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>30</y>
       <width>16</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <zorder>radioButtonB</zorder>
    <zorder>radioButtonD</zorder>
    <zorder>radioButtonGS</zorder>
    <zorder>radioButtonG</zorder>
    <zorder>coverUp1</zorder>
    <zorder>coverUp2</zorder>
    <zorder>coverUp3</zorder>
    <zorder>coverUp4</zorder>
   </widget>
   <widget class="QWidget" name="widget_2" native="true">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>460</y>
      <width>391</width>
      <height>81</height>
     </rect>
    </property>
    <widget class="QRadioButton" name="radioButtonOrg">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>0</y>
       <width>91</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QRadioButton
            {
            font : 32px 'Bebas Neue';
            color : rgb(64, 68, 72);
            }
QRadioButton::checked
            {
            color : rgb(255, 114, 0);
            }</string>
     </property>
     <property name="inputMethodHints">
      <set>Qt::ImhNone</set>
     </property>
     <property name="text">
      <string>.org</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
     <property name="autoExclusive">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButtonGov">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>0</y>
       <width>71</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QRadioButton
            {
            font : 32px 'Bebas Neue';
            color : rgb(64, 68, 72);
            }
QRadioButton::checked
            {
            color : rgb(255, 114, 0);
            }</string>
     </property>
     <property name="inputMethodHints">
      <set>Qt::ImhNone</set>
     </property>
     <property name="text">
      <string>.gov</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
     <property name="autoExclusive">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButtonCom">
     <property name="geometry">
      <rect>
       <x>-20</x>
       <y>0</y>
       <width>91</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QRadioButton
            {
            font : 32px 'Bebas Neue';
            color : rgb(64, 68, 72);
            }
QRadioButton::checked
            {
            color : rgb(255, 114, 0);
            }</string>
     </property>
     <property name="inputMethodHints">
      <set>Qt::ImhNone</set>
     </property>
     <property name="text">
      <string>.com</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
     <property name="autoExclusive">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButtonEdu">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>0</y>
       <width>91</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QRadioButton
            {
            font : 32px 'Bebas Neue';
            color : rgb(64, 68, 72);
            }
QRadioButton::checked
            {
            color : rgb(255, 114, 0);
            }</string>
     </property>
     <property name="inputMethodHints">
      <set>Qt::ImhNone</set>
     </property>
     <property name="text">
      <string>.edu</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
     <property name="autoExclusive">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButtonNet">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>0</y>
       <width>101</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QRadioButton
            {
            font : 32px 'Bebas Neue';
            color : rgb(64, 68, 72);
            }
QRadioButton::checked
            {
            color : rgb(255, 114, 0);
            }</string>
     </property>
     <property name="inputMethodHints">
      <set>Qt::ImhNone</set>
     </property>
     <property name="text">
      <string>.net</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
     <property name="autoExclusive">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLabel" name="coverUp1_3">
     <property name="geometry">
      <rect>
       <x>136</x>
       <y>0</y>
       <width>21</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="coverUp1_5">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>16</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="coverUp1_2">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>0</y>
       <width>16</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <zorder>coverUp1_3</zorder>
    <zorder>coverUp1_5</zorder>
    <zorder>coverUp1_2</zorder>
    <zorder>radioButtonGov</zorder>
    <zorder>radioButtonOrg</zorder>
    <zorder>radioButtonNet</zorder>
    <zorder>radioButtonEdu</zorder>
    <zorder>radioButtonCom</zorder>
   </widget>
   <zorder>superSearcher_5</zorder>
   <zorder>widget</zorder>
   <zorder>superSearcher</zorder>
   <zorder>queryInput</zorder>
   <zorder>pushButton</zorder>
   <zorder>settings</zorder>
   <zorder>darkMode</zorder>
   <zorder>historySection</zorder>
   <zorder>tabsSection</zorder>
   <zorder>errorText</zorder>
   <zorder>widget_2</zorder>
   <zorder>label</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>810</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
